### DISCUSSION TITLE
Add <form> as action for LinkColumn

### DISCUSSION THREAD

**Discussion by YIPZIYAN** [rappasoft/laravel-livewire-tables#988] [Q&A]
I have created a datatable as index for my model.
And to allow user interaction with the model, I wanted to add some action button 
Something like this 
![image](https://user-images.githubusercontent.com/103913961/208579724-69a0ada4-bbf6-45b5-8795-71cc88ddaccf.png)
But in the delete buttons is in a form with DELETE request

Is it possible to do the same for the Link Column. Instead of a link a form is generated?
![image](https://user-images.githubusercontent.com/103913961/208579546-fc31f348-b1b0-4d68-9778-c225491ff34e.png)

**Comment by rappasoft** [comment]
I would just use a view column or format column and pass a view that way you can do whatever you want.

**Reply by YIPZIYAN** [reply]
Can I have some example of it?

**Comment by rappasoft** [comment]
https://rappasoft.com/docs/laravel-livewire-tables/v2/columns/available-methods#content-using-a-view

**Reply by YIPZIYAN** [reply]
Thanks for the link!
How to make a view column you mentioned above?
